app/controllers/scans_controller.rb

```diff
@@ -35,7 +35,7 @@ def current_request
 
   def illiad_query(scan)
     illiad_params = {
-      'rft.genre': 'scananddeliverArticle',
+      'Action': '10', 'Form': '30', 'rft.genre': 'scananddeliverArticle',
```

Comment:
Perhaps add these to the the test @ https://github.com/sul-dlss/sul-requests/blob/master/spec/controllers/scans_controller_spec.rb#L91-L97 so we can make sure we assert they're important.